src/sync-endpoint-cloud-setup.rst

```diff
@@ -259,29 +259,44 @@ Setting up a virtual machine
   .. image:: /img/setup-azure/azure3.png
    :width: 600
 
-2. Create a new resource group to attach to this virtual machine by clicking on :guilabel:`Create new`. Additionally, enter a name for the virtual machine and make sure that *Ubuntu Server 18.04 LTS* is selected for the image name as Ubuntu Server 20.04 LTS has not been tested yet.
+2. On the **Basics** pane, select your Azure subscription, create a new resource group to attach to the virtual machine by clicking on :guilabel:`Create new`. Additionally, enter a name for the virtual machine, select a region and select *Ubuntu Server 22.04 LTS* as the image.
 
   .. image:: /img/setup-azure/azure4.png
     :width: 600
 
+  .. image:: /img/setup-azure/azure5.png
+    :width: 600
+
 3. Scroll down and select your authentication type. We highly recommend that use an SSH key for authentication. Copy and paste your SSH key username, and the key itself.
 
   Use the `following resource <https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/create-with-openssh/>`_ to learn more about creating an SSH key.
 
   Alternatively, Azure now provides an option to automatically generate an SSH key pair (As highlighted in the figure below). This key .pem file can then be directly downloaded to the user's computer for future use to connect to the virtual machine.
 
-  .. image:: /img/setup-azure/azure5.png
+  .. image:: /img/setup-azure/azure6.png
     :width: 600
 
-4. Click the **Advanced** tab at the top and copy and paste the contents from the :download:`cloud_init_AZURE.yml </files/cloud_init_AZURE.yml>` file into the *Custom data* box. Finally, click :guilabel:`Review + create` to actually create the machine.
-If you had generated the SSH key pair through Azure automatic generate key pair option, then it now gives you a prompt to download the key (.pem) file. It is important to download it and remember the path to this file in your computer for connecting to virtual machine later.
+4. Go to the **Networking** tab, enable Public inbound ports, and select HTTP (80), HTTPS (443), and SSH (22) from the dropdown menu.
 
-  .. image:: /img/setup-azure/azure6.png
+  .. image:: /img/setup-azure/azure7.png
     :width: 600
 
-5. In order to modify the firewall settings and change the type of incoming traffic we want to allow, we need to modify the **Networking** settings of our VM. Navigate to this section and then add an inbound security rule that matches the rule below. Leave the outbound rules as-is.
+5. Click the **Advanced** tab at the top and copy and paste the contents from the :download:`cloud_init_AZURE.yml </files/cloud_init_AZURE.yml>` file into the *Custom data* box.
```

Comment:
As Waylon mentioned probably need to explain a bit what this does. "This file will..."

Reply:
Done